.github/ISSUE_TEMPLATE/2-deprecate-sgid-dataset.md

```diff
@@ -29,18 +29,33 @@ The replaced data is still accessible via our shelved policy in AGOL (a link to
 1. _Check [x] the box when the task is completed and add the date of completion._
 1. _~Strike~ out all items that do not apply._
 
-- [ ] Remove data from the Internal SGID (name, completed: `2021/00/00`)
+### Soft Delete
+
+_The purpose of the soft delete is to ensure that all of our users and applications have gracefully migrated off of the dataset. Soft deletes will remain in effect for 14 days. During this time, we will have the ability to restore the dataset to its original SGID offering(s). After these 14 days, the item is then ready for a hard delete._
+
+Note: If this dataset is being replaced, then wait until the new data is publicly available before completing these steps:
+
 - [ ] Mark ArcGIS Online item as deprecated in preparation for future deletion (name, completed: `2021/00/00`)
   - [ ] Change `Authoritative` field to `d` in `SGID.META.AGOLItems` to set the `Deprecated` AGOL flag (name, completed: `2021/00/00`)
   - [ ] Add note to ArcGIS Online description noting when layer will be deleted and any replacement layer (name, completed: `2021/00/00`)
-- [ ] Delete ArcGIS Online item (name, completed: `2021/00/00`)
 - [ ] Unshare item from Open Data (name, completed: `2021/00/00`)
-  - currently achieved by either deleting the AGOL item, changing the `AGOL_ITEM_ID` field in `SGID.META.AGOLItems` to something other than an Item ID and manually removing the item from all SGID groups, or shelving the data.
+- [ ] Remove the SGID Index item (name, completed: `2021/00/00`)
+- [ ] Update the `AGOL_ITEM_ID` field in the `SGID.META.AGOLItems` table to a non-GUID text (ie: `Removed from AGOL` or `Replaced by abc from xyz`). Allow this updated text to persist though one run of [Auditor](https://github.com/agrc/auditor) - currently, Auditor runs daily at 5:00am (name, completed: `2021/00/00`)
+  - [ ] After one successful run of Auditor, remove the row from the `SGID.META.AGOLItems` table. This will trigger the removal of this item in Open SGID (name, completed: `2021/00/00`)
+
+### Hard Delete
+
+_Hard deletes are final. It is recommend to complete the the soft delete process before moving on to these steps. If you decide to skip the soft delete, note that you will need to incorporate some of those steps here._
+
+- [ ] Remove data from the Internal SGID (name, completed: `2021/00/00`)
+  - Deprecated database layers can be backup up on Google Drive > AGRC Share > Team Projects > SGID > [deprecated layers]
+- [ ] Delete ArcGIS Online item (name, completed: `2021/00/00`)
+  - currently achieved by either deleting the AGOL item, changing the `AGOL_ITEM_ID` field in `SGID.META.AGOLItems` to something other than an Item ID and manually removing the item from all SGID groups, or shelving the data. 
```

Comment:
Do we have something in place that automatically deletes AGOL items? Auditor will ignore an item if `AGOL_ITEM_ID` is not a valid GUID, but it won't delete the item.

Reply:
I do not think we have any deletion automation.

Reply:
Yeah, I agree. I was confused by the `by either....` list, which could be interpreted as something will automagically delete the item when it the `AGOL_ITEM_ID` field changes.

Reply:
i  updated the wording to make it more clear.